packages/donation-form/src/donation-form.ts

```diff
@@ -187,6 +195,12 @@ export class DonationForm extends LitElement {
     `;
   }
 
+  private get formSectionNumberingStart(): number {
+    // In "Button" mode, the frequency selector is section 1,
+    // but in "Checkbox" mode, it's just a small checkbox below the amount selector
+    return this.frequencySelectionMode === EditDonationFrequencySelectionMode.Button ? 3 : 2;
```

Comment:
whew this one was tough for me, even with very explicit variable name 😅 - ok, so, just to make sure I see what's happening:

`EditDonationFrequencySelectionMode.Button` is where we select `Once | Monthly`
- it's usually a button, which adds an extra step to the donation form flow.  so it' starts at 3 bc:
```
1. Frequency
2. amount
3. payment type
4. Contact
5. Donate
```
so now the new checkbox view shifts steps to:
```
1. amount  + Frequency checkbox
2. payment type
3. Contact
4. Donate
```
<img width="231" alt="image" src="https://user-images.githubusercontent.com/7840857/133162721-c8dc760d-b962-4c08-91c5-d3afcbe7719c.png">

Reply:
Q: will this upcoming test determine the default view?

Reply:
_sigh_ Yeah this wasn't my first choice. My mind wandered into creating a higher-level "numbering manager" to take care of the numbering depending on the absolute count and not based on a mode being set. It might still be worth pursuing that since we may be modifying the sections in the future, I just didn't have time for this ABTest.

But yes, you have it correct and I'm going to put your example in the comment since it's kind of hard to understand. 😄

Reply:
I added a big ole jsdoc for this method to help explain it until we get the real deal in place